src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
 // write code here
+
+const list = document.querySelector('ul');
+const items = [...list.children];
+
+function sortList(elements) {
+  [...elements]
+    .sort((a, b) => {
+      const salaryA = formatedSalary(a.getAttribute('data-salary'));
+      const salaryB = formatedSalary(b.getAttribute('data-salary'));
+
+      return salaryB - salaryA;
+    })
+    .forEach((person) => {
+      list.appendChild(person);
+    });
+}
+
+function formatedSalary(salary) {
+  return Number(salary.replace(/,/g, '.').replace(/[$]/g, ''));
+}
+
+function getEmployees(elements) {
+  return elements.map((person) => ({
+    name: person.textContent.trim(),
+    position: person.getAttribute('data-position'),
+    salary: `$` + formatedSalary(person.getAttribute('data-salary')).toFixed(3),
```

Comment:
The salary is being formatted with three decimal places. Ensure this is the intended format. If not, adjust the `toFixed` method accordingly.